core/Cargo.toml

```diff
@@ -65,6 +65,10 @@ sp-core = { workspace = true, optional = true }
 sp-runtime = { workspace = true, optional = true }
 tracing = { workspace = true, default-features = false }
 
+# AccountId20
+keccak-hash = { workspace = true}
```

Comment:
quite annoying to add this dependency to core but no way around it I guess without feature gating it...